### DISCUSSION TITLE
How to apply a repeating pattern/texture to a polygon

### DISCUSSION THREAD

**Discussion by lagouyn** [rexrainbow/phaser3-rex-notes#422] [Q&A]
Hello.

I'm curious if the Rex Plugins provide a way to apply a repeating pattern/texture to a polygon.

Or, does anyone have any suggestions for doing this directly with Phaser 3? I've made some attempts with a mask, but nothing so far has worked well for me.

Thanks.
-lagouyn

**Comment by rexrainbow** [comment]
Repeating pattern/texture could be created by [tileSprite](https://labs.phaser.io/index.html?dir=game%20objects/tile%20sprite/&q=), then apply geometry mask created from Graphics with polygon
